x/lockup/keeper/lock.go

```diff
@@ -718,6 +718,8 @@ func (k Keeper) unlock(ctx sdk.Context, lock types.PeriodLock) error {
 		return err
 	}
 
+	k.hooks.OnTokenUnlocked(ctx, owner, lock.ID, lock.Coins, lock.Duration, lock.EndTime)
```

Comment:
Is there a reason behind moving the hook? (I don't really have an opinion, just curious)

Also perhaps worth us coming back and renaming the hook to be `AfterTokenUnlocked`

Reply:
So there's logic added to hooks that uses lock reference to update periodLockReward before we delete the lock!

Reply:
Oer renaming the hooks, do you think it's a better decision to do it in a separate PR or straight up ahead on this one?